src/engraving/dom/tremolotwochord.h

```diff
@@ -152,7 +152,17 @@ class TremoloTwoChord final : public EngravingItem
     void computeShape();
     //! -----------------
 
-    std::shared_ptr<rendering::dev::BeamTremoloLayout> layoutInfo;
+    void setLayoutInfo(std::shared_ptr<rendering::dev::BeamTremoloLayout> info)
+    {
+        m_layoutInfo = info;
+    }
+
+    std::shared_ptr<rendering::dev::BeamTremoloLayout> layoutInfo() const
+    {
+        return m_layoutInfo;
+    }
+
+    std::shared_ptr<rendering::dev::BeamTremoloLayout> m_layoutInfo;
```

Comment:
Was this intended to be made private, or is it fine like this?

Reply:
After a few PRs this field will be removed, so it doesn't matter